src/types.rs

```diff
@@ -293,7 +295,38 @@ pub struct RuntimeConfig {
 	/// The maximum number of provider records for which the local node is the provider. (default: 1024).
 	pub max_kad_provided_keys: u64,
 	/// Avail account secret key. (default: None)
-	pub avail_secret_key: Option<String>,
+	pub avail_secret_key: Option<AvailSecretKey>,
+}
+
+#[derive(Deserialize, Clone)]
+#[serde(try_from = "String")]
+pub struct AvailSecretKey(pub Pair);
+
+impl Serialize for AvailSecretKey {
+	fn serialize<S>(&self, serializer: S) -> std::result::Result<S::Ok, S::Error>
+	where
+		S: serde::Serializer,
+	{
+		serializer.serialize_bytes(&self.0.to_raw_vec())
+	}
+}
+
+impl std::fmt::Debug for AvailSecretKey {
+	fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+		f.debug_tuple("AvailSecretKey")
+			.field(&self.0.to_raw_vec())
+			.finish()
+	}
+}
```

Comment:
I think it is a bad idea to implement Debug for it. The best would be to skip this field in other structures. There is a reason why substrate guys didn't derive Debug for secret key, righ.

Reply:
Sure, I have to check it once how to avoid it, but didn't wanted to left uncommited.

Reply:
So, I have removed field from serialization since default is none anyway, and for debugging, i have left public key which could be useful for identification.